web/modules/custom/dbcdk_community/src/Plugin/Block/ProfilesBlock.php

```diff
@@ -208,6 +250,48 @@ public function build() {
   }
 
   /**
+   * Get all quarantined profiles.
+   *
+   * @param array $profiles_filter
+   *   An array containing filter arguments.
+   *
+   * @throws \DBCDK\CommunityServices\ApiException
+   *   Throws API Exception if any of the calls to the services fails.
+   *
+   * @return \DBCDK\CommunityServices\Model\Profile[] $profiles
+   *   An array of Profile objects with an active quarantine.
+   */
+  protected function getQuarantinedProfiles(array $profiles_filter) {
+    // Fetch all quarantines that are active from the moment of the request.
+    $quarantined_filter = [
+      'where' => [
+        'end' => [
+          'gt' => date('Y-m-d\TH:i:s.000\Z'),
```

Comment:
Could we use one of [the predefined formats](http://php.net/manual/en/class.datetime.php#datetime.constants.types) here?